_posts/2016-10-03-what-i-wish-i-knew.md

```diff
@@ -0,0 +1,96 @@
+---
+author: David Hatanian
+comments: true
+date: 2016-10-03 10:10:00 +00:00
+layout: post
+asset-type: post
+slug: what-i-wish-i-knew-earlier
+title: What I wish I knew when I started as a software developer
+image:
+    src: /assets/img/custom/blog/2016-10-03-what-i-wish-i-knew/thumbnail.jpg
+attribution:
+    text: Young poplars on Westrop Farm (licensed under Creative Commons)
+    href: http://www.geograph.org.uk/photo/2469549 
+tags:
+- craftsmanship
+- practices
+- productivity
+---
+
+Having worked in the software engineering field for 6 years, I sometimes pause and consider how much I have learned since I started my career. One thing that strikes me is how many things I would want to tell just-out-of-college me if I had the occasion.
+
+While I don’t have that capability, I hope that the few items below will be useful to people on their way out of college and into the software industry.
+
+#Keep on learning#
+You just obtained your diploma, are overwhelmed by a sense of accomplishment and now feel the desire to prove yourself to your new colleagues? That is great, congratulations and good luck, but keep in mind that this is just the beginning of your learning.
+
+You will find soon that you need to keep learning new technologies, new methods, new techniques, new languages. Start as soon as possible, because learning in an unsupervised way (no syllabus, no teacher, no final exam) is a skill that you need to hone.
+
+Learn things outside your comfort zone. If you work on a dynamic language, learn a statically typed one. Use VIM or Emacs if you are used to an IDE, etc.
+
+##Find a mentor##
+I just said that there are no teachers after college, but that is not entirely true. Nobody will seat in a classroom with you and go over a predefined lesson. But it is still extremely beneficial to find one more experienced person that can guide you.
+
+[Sandro’s book “The Software Craftsman”](https://www.goodreads.com/book/show/23215733-the-software-craftsman) describes how a mentor is beneficial to a new software developer at great lengths.
+
+Depending on the company you are working with, you might be provided with a mentor, or someone will come to you and spontaneously offer their help. Sometimes you will need to look for them. If you cannot find them amongst your colleagues, look for them in community events for example.
```

Comment:
Sometimes you'll need to approach someone at your company... and sometimes you have looked within your company and found nothing. So you need to search in the community.

Looks like the sentence is misleading. How could I phrase this better?